setup.cfg

```diff
@@ -1,7 +1,14 @@
-[nosetests]
-nocapture=1
-cover-package=zodbpickle
-cover-erase=1
-with-doctest=0
-where=src/zodbpickle
+[bdist_wheel]
+universal = 0
 
+[coverage:run]
```

Comment:
Personally I prefer to keep coverage configuration in .coveragerc, but my reasons are pretty trivial. First, `coveralls` passes `".coveragerc"` as the 'config_file' option to `coverage`, as perhaps other tools might. In the long-long ago, `coverage` would only look in that passed file for its config, so it was just generally simpler to keep it there. Second, I find it easier to cargo-cult around a .coveragerc file from project to project---it's just a 'cp' if the file specifies `source = src/`---than it is a 'setup.cfg' if the destination project already has a 'setup.cfg', which is likely.